framework/test/unit/modules/dpos_v2/module.spec.ts

```diff
@@ -1020,7 +1020,7 @@ describe('DPoS module', () => {
 
 				const missedBlocks: Record<string, number> = {};
 				// Make every delegate miss its block-slot except start and end slots
-				for (let i = 1; i < 102; i += 1) {
+				for (let i = 0; i < 102; i += 1) {
```

Comment:
I think the comment and the logic doesn't match here?

Reply:
It actually matches, that confused me in the first place. The forging starts from last round's last block which is omitted as start slot. So if we start from current round's beginning block we omit start slot if we think context is bounded to `getValidatorsBetweenTimestamps`, it's a matter of context. I choosed this context because `missedBlocks` is related to `getValidatorsBetweenTimestamps`. Do you have another advice on that?

Reply:
I think 
> `missedBlocks` is related to `getValidatorsBetweenTimestamps`

this assumption is correct, but the loop executes 102 times where it should be 101 times if we dont include start and end?

Reply:
Start: previous round's last block
End: current round's last block
= 104 blocks(1 + 103). Omit `Start` and `End`, you have 102 blocks. `Start` and `End` may point to same delegate since their positions are same, at least in this test case, so we set `consecutiveMissedBlocks` to 1 for other 102 delegates. This was my logic.

Reply:
Ah I think I understand. 
> Start: previous round's last block
End: current round's last block

I think it was originally first block of the current round to the  current block, but i think the result is the same, so it's ok